### ISSUE TITLE
New v10 version

### ISSUE THREAD

**Issue by HippyCraig** [AmazingVanish/apsj#10]
Is this still being maintained?  I was testing your dev version for v10 I was wondering when that would make it to the main repo?

**Comment by AmazingVanish** [comment]
@HippyCraig Yes and no. I have updated the module beyond what is available online and I am working with IronMonk to merge it into Monk's Enhanced Journal. It will no longer be available separately.

**Comment by HippyCraig** [comment]
ok, thanks for hte update I like monks stuff but its not really a semless integration for my needs with the new Journals in v10.  I may revisit that in the future but the work both of you do is great!
